adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs

```diff
@@ -98,6 +98,10 @@ public PlatformParameters(Activity callerActivity, bool useBroker, PromptBehavio
         /// </summary>
         public PromptBehavior PromptBehavior { get; set; }
 
+        /// <summary>
+        /// Used to set embedded 
```

Comment:
>Used to set embedded  [](start = 12, length = 21)

half sentence?

Reply:
yes...that's what happens when trying to finish before the forced update :)

---
In reply to: [190771933](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1069#discussion_r190771933) [](ancestors = 190771933)